### DISCUSSION TITLE
TOC 데스크탑에서 안나오는 에러입니다.

### DISCUSSION THREAD

**Discussion by honge7694** [pronist/hELLO#71] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: [windows]
- 브라우저: [chrome]
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [3.6.1]

**설명**

TOC의 기능이 정상 작동되지 않습니다. 다른 블로그의 글은 잘 나타나는데 제껀 안나오네요..
TOC 사용 설정도 해놓았습니다.

**스크린샷 및 링크**
toc 설정 확인
![image](https://user-images.githubusercontent.com/76715487/159550575-ff9b5f8b-61a1-4911-b288-fac818a96a49.png)

TOC 적용 안됨 확인.
![image](https://user-images.githubusercontent.com/76715487/159550703-04b05a2c-6b66-43a9-ac48-9563550a6b80.png)

해당 게시글입니다
https://bwhong.tistory.com/4?category=926088

**Comment by pronist** [comment, marked as answer]
이 질문은 중복입니다 #51 을 참고해주세요. 

그 외에는 [스킨 배포페이지 - 트러블슈팅 - FAQ](https://pronist.tistory.com/5#FAQ) 에도 설명되어있으니 참고부탁드립니다.  
TOC 는 가로 1820px 이하에선 나오지 않습니다.

![1](https://user-images.githubusercontent.com/25624965/159600515-8d88f911-fbae-4487-b67a-7f0a77e7aefa.PNG)

**Comment by honge7694** [comment]
아 감사합니다! 저렇게 확인을 못헀네요!!
